woocommerce/woocommerce
# Remove padding for the products query when background is added

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes the margin that is automatically added to the product query block when a background color is set.

Closes https://github.com/woocommerce/woocommerce/issues/42226



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post and add the "Products (Beta)" block.
2. Go to the "List view".
3. Click on the "Product Template" inner block.
4. Change the background color from the sidebar.
5. Check the background color is applied and there are no changes in margins.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Remove padding for the products query when background is added
#### Comment 

</details>
